vtm-playground/src/org/oscim/test/gdx/poi3d/Gdx3DTest.java

```diff
@@ -57,6 +58,13 @@ public void createLayers() {
 
         mMap.layers().add(new Poi3DLayer(mMap, mMapLayer));
 
+        /*
+        // Add car model to "Neue Elbbrücke" in Hamburg.
+        GdxModelLayer gdxModelLayer = new GdxModelLayer(mMap);
+        mMap.layers().add(gdxModelLayer);
+        gdxModelLayer.addModel(VtmModels.CAR, 53.53202, 10.02599, 300f);
+        */
```

Comment:
Is it needed?

Reply:
I thought it would be good to have an example... I comment it, cause in most cases it won't be needed. Your choice